### PR TITLE
Add ellipse drawing functions

### PR DESCRIPTION
**Pull request by huderlem**
Add hollow and filled ellipse drawing.
Part of #151

### CODE REVIEW

**Comment by theotherphil** [comment]
I've commented on a couple of very minor issues, but otherwise LGTM.

**Comment by huderlem** [comment]
I don't see your comments anywhere in this Pull Request. They're usually inlined with the file changes, correct?

**Comment by theotherphil** [comment]
Oops! They should be visible now.

File: src/drawing.rs [hunk 1]
```diff
@@ -274,6 +274,126 @@ pub fn draw_filled_rect_mut<I>(image: &mut I, rect: Rect, color: I::Pixel)
     }
 }
 
+/// Draw as much of an ellipse as lies inside the image bounds.
```
Comment (huderlem) [inline src/drawing.rs]: Done.

File: src/drawing.rs [hunk 2]
```diff
@@ -274,6 +274,126 @@ pub fn draw_filled_rect_mut<I>(image: &mut I, rect: Rect, color: I::Pixel)
...
+pub fn draw_hollow_ellipse<I>(image: &I,
+                             center: (i32, i32),
+                             width_radius: i32,
+                             height_radius: i32,
+                             color: I::Pixel) -> VecBuffer<I::Pixel>
+    where I: GenericImage,
```
Comment (huderlem) [inline src/drawing.rs]: Done.

**Comment by theotherphil** [comment]
Great, thanks!

**Pull request merged**